src/harness/util.py

```diff
@@ -347,12 +347,11 @@ def convertRequestToRequestWithCpiSignature(private_key, cpi_id,
 def computePropagationAntennaModel(request):
```

Comment:
I think this belongs in the PAT file since it will be used only in that test case. We have a convention of only adding items to this file if they will be used by multiple test cases across multiple files, and I don't see that being true in this case.